src/types/redis_hash.cc

```diff
@@ -250,10 +254,14 @@ rocksdb::Status Hash::MSet(const Slice &user_key, const std::vector<FieldValue>
   auto batch = storage_->GetWriteBatchBase();
   WriteBatchLogData log_data(kRedisHash);
   batch->PutLogData(log_data.Encode());
+  std::unordered_set<std::string> field_set;
+  for (int32_t i = static_cast<int32_t>(field_values.size() - 1); i >= 0; i--) {
```

Comment:
Could we use `size_t` instead of `int32_t` here and omit the `static_cast` usage?

Reply:
`for (auto it = field_values.rbegin(); it != field_values.rend(); it++)`

Reply:
`size_t` is not ok, it is unsigned type,  always `>= 0`.

`for (auto it = field_values.rbegin(); it != field_values.rend(); it++) ` could be a good choice.